.site/_layouts/main.njk

```diff
@@ -6,6 +6,7 @@
 	<link rel="stylesheet" href="/css/layout.css?{{ "" | cachebuster }}">
 	<link rel="stylesheet" href="/css/nav.css?{{ "" | cachebuster }}">
 	<link rel="stylesheet" href="/css/style.css?{{ "" | cachebuster }}">
+	<link rel="stylesheet" href="https://unpkg.com/prismjs@1.24/themes/prism.css">
```

Comment:
I would prefer to vendor this.  There's not a lot of bytes in there, and it'll make me happier when I'm hacking on a train.

Reply:
Did so.